_data/guides.yml

```diff
@@ -8,10 +8,12 @@ toc:
   section:
   - title: What is Kubernetes?
     path: /docs/whatisk8s/
+  - title: Installing Kubernetes Easily on Linux
```

Comment:
"Easily" verges on marketing speak and is generally not used in technical documentation. "Installing Kubernetes on Linux with kubeadm" is probably better.